src/main/scala/kanaloa/reactive/dispatcher/Dispatcher.scala

```diff
@@ -102,12 +125,14 @@ case class PushingDispatcher(
 )
   extends Dispatcher {
 
-  protected lazy val queueProps = Queue.withBackPressure(
-    settings.bufferHistory,
-    settings.backPressure.getOrElse(Dispatcher.defaultDispatcherConfig().as[BackPressureSettings]("backPressure")),
-    WorkSettings(),
-    metricsCollector
-  )
+  protected lazy val queueProps = settings.backPressure.map(
+    Queue.withBackPressure(
+      settings.dispatchHistory,
+      _,
+      WorkSettings(),
+      metricsCollector
+    )
+  ).getOrElse(Queue.default(settings.dispatchHistory, WorkSettings(), metricsCollector))
```

Comment:
same here, a `match` might be clearer
https://github.com/iheartradio/kanaloa/pull/28/files#r44564783